design2.css

```diff
@@ -0,0 +1,136 @@
+body {
+	background-color: #008800;
+	font-family: Verdana, Arial, Helvetica, sans-serif;
+	text-align: center;
+}
+* {
+	margin: 0px;
+	padding: 0px;
+}
+html {height:100%; margin-bottom:1px;} /* Forces vertical scroll bar to be visible */
+#wrapper {
+	width: 900px;
+	margin-right: auto;
+	margin-left: auto;
+	background-image: url(images/Column1.jpg);

```

Comment:
the `url(images/Column1.jpg); ` is missing a quote around some browsers aren't capable of parsing this again some recent one does.  we can have this rewritten as `url('images/Column1.jpg'); `